libuuu/fastboot.cpp

```diff
@@ -459,6 +457,15 @@ int FBFlashCmd::parser(char *p)
 		m_raw2sparse = true;
 		m_partition = get_next_param(subcmd, pos);
 	}
+	else if (m_partition == "-S")
+	{
+		m_partition = get_next_param(subcmd, pos);
+		bool conversion_success = false;
+		m_sparse_limit = str_to_uint64(m_partition, &conversion_success);
+		if (!conversion_success)
+			return -1;
+		m_partition = get_next_param(subcmd, pos);
+	}
 
```

Comment:
Command line parser should be in int FBFlashCmd::parser(char *p) 

Transport only do action according to parser result.

Reply:
yes. It is there.

Reply:
sorry, it is github show problem.

A small problem need be fixed.